olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -179,10 +181,14 @@ class AuthenticationClient::Impl final {
   void SetTaskScheduler(std::shared_ptr<TaskScheduler> task_scheduler);
 
  private:
+  using TimeResponse = client::ApiResponse<time_t, client::ApiError>;
+  using TimeCallback = std::function<void(TimeResponse)>;
```

Comment:
Would it make sense to move them closer to the usage in the SignInClient?

Reply:
What do you mean by that? they are near GetTimeFromServer() method and are used in declaration of that method.

Reply:
Pardon me. Didn't notice that `TimeResponse` in is the declaration down there.